tests/test_abbenay_client_factory.py

```
@@ -0,0 +1,202 @@
+            cfg = resolve_abbenay_tls_config("abbenay:50057")
+        assert cfg.enabled is False
+
+    def test_auto_tls_when_default_ca_exists(self, monkeypatch: pytest.MonkeyPatch) -> None:
+        """Podman shared runtime CA enables TLS without explicit env.
+
+        Args:
+            monkeypatch: Pytest monkeypatch fixture.
+        """
+        monkeypatch.delenv("APME_ABBENAY_TLS", raising=False)
+        monkeypatch.delenv("APME_ABBENAY_CA_CERT", raising=False)
+        ca_path = "/tmp/abbenay-run/abbenay/tls/ca.crt"
```

Ruff (0.16.0)

[error] Probable insecure usage of temporary file or directory: "/tmp/abbenay-run/abbenay/tls/ca.crt"

(S108)